R/simple_ensemble.R

```diff
@@ -56,27 +56,27 @@ simple_ensemble <- function(model_outputs, weights = NULL,
     cli::cli_abort(c("x" = "{.arg model_outputs} must be a `data.frame`."))
   }
 
+  if (isFALSE("model_out_tbl" %in% class(model_outputs))) {
+    hubUtils::as_model_out_tbl(model_outputs)
```

Comment:
note that `hubUtils::as_model_out_tbl` does not modify an object in place, so you will need to assign the result of this call to update `model_outputs` here.
```suggestion
    model_outputs <- hubUtils::as_model_out_tbl(model_outputs)
```

Reply:
Additionally -- I believe that this will result in an error if the provided `model_outputs` is a data frame and the provided `output_type_col` and `output_type_id_col` are something other than `"output_type"` and `"output_type_id"` respectively. But I think maybe now that we have a formal `model_out_tbl` object class that standardizes its column names, we should just expect to work with those column names? Let's leave this as is for now and I will file a separate issue.